.storybook/preview.tsx

```diff
@@ -54,6 +54,7 @@ const DocumentationWrapper = ({ children, context }) => {
 };
 
 const preview: Preview = {
+  tags: ['autodocs'],
```

Comment:
According to the change in .storybook/main.ts.
```
docs : {
   autodocs : true 
}
```
is deprecated.
See : https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mainjs-docsautodocs-is-deprecated